articles/active-directory/develop/msal-android-b2c.md

```diff
@@ -3,15 +3,15 @@ title: Azure AD B2C (MSAL Android) | Azure
 titleSuffix: Microsoft identity platform
 description: Learn about specific considerations when using Azure AD B2C with the Microsoft Authentication Library for Android (MSAL.Android)
 services: active-directory
-author: brianmel
+author: iambmelt
```

Comment:
@CelesteDG Not sure if this is correct/advisable, `iambmelt` is my GitHub identity, `brianmel` is my corp alias

Reply:
The requested revision is correct; **author** should be the author's GitHub username. It's how automation assigns pull requests that are updated in the public repository.

Reply:
Thanks @ktoliver -- I'll this one